libs/geography/src/drivers/magento/transforms/responses/country.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+
+import { DaffMagentoSubdivisionTransformer } from './subdivision.service';
+import { MagentoCountry } from '../../models/responses/country';
+import { DaffCountry } from '../../../../models/country';
+
+/**
+ * Transforms magento carts into an object usable by daffodil.
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffMagentoCountryTransformer {
+  constructor(public subdivisionTransformer: DaffMagentoSubdivisionTransformer) {}
+
+  private transformSubdivisions(country: MagentoCountry) {
+    return {
+      subdivisions: country.available_regions.map(region => this.subdivisionTransformer.transform(region))
+    }
+  }
+
+  /**
+   * Transforms the MagentoCountry from the magento country query into a DaffCountry.
+   * @param country the country from a magento country query.
+   */
+  transform(country: MagentoCountry): DaffCountry {
+    return country ? {
```

Comment:
Why the conditional?

Reply:
See [this discussion](https://github.com/graycoreio/daffodil/pull/696#issuecomment-598280442) between Luke and I. Basically null checking has to happen somewhere and inside the transformer seems like the best place. I made a habit of checking for `null` inside transformers since TS does not prevent calling `transform(null)`.